tests/Integration/CompletorTestCase.php

```diff
@@ -39,7 +39,11 @@ protected function assertComplete(string $source, array $expected, bool $isCompl
 
         $this->assertCount(count($expected), $suggestions);
         foreach ($expected as $index => $expectedSuggestion) {
-            $this->assertArraySubset($expectedSuggestion, $suggestions[$index]->toArray());
+            $actual = $suggestions[$index]->toArray();
+            $this->assertArraySubset($expectedSuggestion, $actual);
+            if (array_key_exists('snippet', $expectedSuggestion) === false) {
```

Comment:
To cover the snippet creation I had to extend the test. My first attempt was to just test for equality with the expected result. But the other tests don't pass the entire result array. That is the reason why I just test snippets here.